package.json

```diff
@@ -94,6 +98,7 @@
       "registry": "https://www.npmjs.com/package/@mitre/saf",
       "message": "<%= config.name %> update available from <%= chalk.greenBright(config.version) %> to <%= chalk.greenBright(latest) %>."
     },
+    "topicSeparator": ":",
```

Comment:
`":"` is the default, but with v2 this can be changed and specified to be a space `" "`.

Reply:
@aaronlippold @ejaronne Do we want to switch this? It would mean breaking changes for every implementation of the CLI currently unless there's a way to support both

Reply:
space seems most natural to me

Reply:
it is not a breaking change according to the migration docs:

> NOTE: Using colons always works, even if you set the topicSeparator to spaces. This means that you can enable spaces in your CLI without introducing a breaking change to your users.

Can @em-c-rod double check to confirm that this is actually the case?

Reply:
I changed it to space, tested locally, and added a test to the suite that uses the space instead of the ":" for a separator.